src/i18n/Localizable.xcstrings

```diff
@@ -19596,7 +19699,8 @@
       }
     },
     "Restart" : {
-      "extractionState" : "migrated",
+      "comment" : "is this one used as verb or noun?",
```

Comment:
"Restart" is an action, so a verb I would say

Reply:
deleted comment. if "Restart" is an action, leave this as-it-is(noun) would be better, because in Korean noun or gerund is usually used to implement actions